test/DynamoCoreTests/CustomNodes.cs

```diff
@@ -750,6 +750,39 @@ public void TestCustomNodeDefaultValue()
         }
 
         [Test]
+        public void TestCustomNodeDefaultValueJson()
+        {
+            // load xml dyf test file
+            var dynFilePath = Path.Combine(TestDirectory, @"core\CustomNodes\CNDefault.dyf");
+            OpenModel(dynFilePath);
+
```

Comment:
@mjkkirschner I added this xml test file because most of the existing custom node test files contain default values that are either integers or doubles.  I wanted to verify geometry defaults were producing the appropriate strings as well.